### ISSUE TITLE
Ошибка при быстром добавлении блоков или вставке из буфера

### ISSUE THREAD

**Issue by MantisReligiosa** [MantisReligiosa/Display-control#20]

**Comment by MantisReligiosa** [comment]
Исправлено начиная с версии 0.0.4
Ссылка на zip: https://drive.google.com/open?id=13fW4jcNgD0CUmX0s34nMoJRf234H78oW
Ссылка на msi: https://drive.google.com/open?id=11XZQwoCcrtGaG-NiK1i7yOiqta-qVqhy

**Comment by Kim134** [comment]
Сейчас вылез БАГ, при добавлении блока ТЕКСТ и наложении на другой блок - после его удаления, курсор мыши становится  "крестиком" как будто на всем поле еще сохранился (хотя удален) блок и при пуске возникает ошибка.

**Comment by MantisReligiosa** [comment]
Да, я тоже такое встречал. Это известный баг в движке браузеров. Ждём, когда его поправят разработчики

**Comment by Kim134** [comment]
done
